### PR TITLE
feat: MET-209 add types for animations

### PR DESCRIPTION
**Pull request by willowish**

### CODE REVIEW

**Comment by willowish** [comment]
Types required for: https://startuphouse.atlassian.net/browse/MET-209

File: src/types/animation/AnimationEventType.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export enum AnimationEventType {
```
Comment (damian08kk) [inline src/types/animation/AnimationEventType.ts:1]: Creating enums in lib doesn't create problems anymore? There was an situation with enums and Dawid changed every enum to type.

Comment (willowish) [reply src/types/animation/AnimationEventType.ts:1]: I used them inside metaverse-front (I've linked local metavrse-lib) and it was fine 🤷

Comment (dawidsdh) [reply src/types/animation/AnimationEventType.ts:1]: Please don't use enums in the library. There is an issue with enums when running test by using the library and the only way to fix it is to use constants. Use enums only on the editor side.

File: src/types/animation/AnimationState.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export enum AnimationState {
```
Comment (dawidsdh) [inline src/types/animation/AnimationState.ts:1]: The same as above

File: src/types/animation/CherryAnimation.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { AnimationDescription } from './AnimationDescription';
+import { AnimationEventType } from './AnimationEventType';
+
+export type CherryAnimation = {
+  getState: () => string;
+  pause: () => void;
+  resume: () => void;
+  play: (config: { id: string }) => void;
+  stop: () => void;
```
Comment (dawidsdh) [inline src/types/animation/CherryAnimation.ts:9]: This method has a parameter. Please add parameters and types to all methods.

File: src/types/animation/AnimationEventType.ts (hunk 1 above)
Comment (dawidsdh) [reply src/types/animation/AnimationEventType.ts:1]: Also there is a folder called constants where we can put this enums after transforming it to consts.

**Review by dawidsdh — CHANGES_REQUESTED** [review]

File: src/types/animation/CherryAnimation.ts (hunk 3 above)
Comment (willowish) [reply src/types/animation/CherryAnimation.ts:9]: image
Well, I don't see any parameters here. How I can get an information about the parameter that you are talking about?

Comment (dawidsdh) [reply src/types/animation/CherryAnimation.ts:9]: Oh sorry my bad, I looked into wrong file. but if you look into the assets in metavrse-lib you will have all the methods. You can follow what we have in assets files.

**Review by dawidsdh — APPROVED** [review]

**Pull request merged**